### DISCUSSION TITLE
How to define "n_last" argument for "termination"

### DISCUSSION THREAD

**Discussion by morceauai** [anyoptimization/pymoo#471] [Q&A] [answered]
Hi guys, 
As mentioned in homepage, "n_last" would be useful to adjust the tolerance of the termination. However it doesn't seem to work.
I also searched for "n_last" in the whole module, still can't find the implement in the code.
Can anyone give a example?

**Comment by blankjul** [comment, marked as answer]
the variable name is now `period` which defines how many last generations shall be used: https://pymoo.org/interface/termination.html

can you please let me know where the documentation still uses n_last and I will fix it.

**Reply by morceauai** [reply]
Thanks Julian. I will try “period”.
It is just on the page of the link you posed.
![IMG_8809](https://github.com/anyoptimization/pymoo/assets/51000305/6b527622-862f-4410-bd29-f8a4268d967c)
I’m not sure if it displays differently on you side.

**Reply by juanlu29** [reply]
I searched on the discussions for this issue as well. I can confirm that n_last still appears as the name of this variable instead of period. Thanks for your work!
